src/style.css

```diff
@@ -2,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar-top {
+  height: 70px;
+  margin-top: 20px;
+}
+
+.search-bar-bottom {
+  height: 42px;
```

Comment:
The same comment about fixing container size applies here for '.search-bar-bottom'. It's better to let the content size dictate the container size.